include/aws/io/uri.h

```diff
@@ -0,0 +1,122 @@
+#ifndef AWS_IO_URI_H
+#define AWS_IO_URI_H
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+#include <aws/common/byte_buf.h>
+
+#include <aws/io/exports.h>
```

Comment:
We've been having every public header include its libname.h, instead of "exports.h"